lib/chef/knife/vsphere_vm_snapshot.rb

```diff
@@ -39,6 +39,12 @@ class Chef::Knife::VsphereVmSnapshot < Chef::Knife::BaseVsphereCommand
          description: 'Indicates whether to start the VM after a successful revert',
          boolean: false
 
+  option :wait,
+         :long => "--wait",
+         :description => "Indicates whether to wait for creation/removal to complete",
+         :boolean => false
+
```

Comment:
A few months ago we started using Ruby 1.9 style hashes, and followed the default Rubocop style guidelines. This means:

* Using `foo: "bar"` instead of `:foo => "bar"`
* Using single quotes for strings where we're not interpolating.

Could you please update this PR with the new style?